hopr/hopr-lib/tests/chain_integration_tests.rs

```diff
@@ -173,11 +173,8 @@ const SNAPSHOT_ALICE_RX: &str = "tests/snapshots/indexer_snapshot_alice_in";
 const SNAPSHOT_BOB_TX: &str = "tests/snapshots/indexer_snapshot_bob_out";
 const SNAPSHOT_BOB_RX: &str = "tests/snapshots/indexer_snapshot_bob_in";
 
+// only working for async-std atm
```

Comment:
Why though? Another PR removes the async-std altogether.

Reply:
We will have to fix this in whatever PRs merges last.